Description: Remove bundled wxsqlite3 from CMakeLists.txt files
Author: James Cowgill <jcowgill@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -543,7 +543,6 @@ if(APPLE)
     add_subdirectory(sqlite3)
 endif()
 
-add_subdirectory(sdk/wxsqlite3)
 add_subdirectory(sdk/wxshapeframework)
 add_subdirectory(sdk/databaselayer)
 add_subdirectory(CodeLite)
@@ -646,9 +645,7 @@ if(APPLE)
     add_dependencies(wxsqlite3 sqlite3lib)
 endif()
 
-add_dependencies(databaselayersqlite wxsqlite3)
-add_dependencies(wxshapeframework wxsqlite3)
-add_dependencies(libcodelite wxshapeframework databaselayersqlite wxsqlite3)
+add_dependencies(libcodelite wxshapeframework databaselayersqlite)
 add_dependencies(plugin libcodelite)
 
 if(NOT SDK_ONLY)
--- a/CodeLite/CMakeLists.txt
+++ b/CodeLite/CMakeLists.txt
@@ -116,7 +116,7 @@ if (UNIX AND NOT APPLE )
                          ${wxWidgets_LIBRARIES} 
                          -L"${CL_LIBPATH}" 
                          ${SQLITE3_LIBRARY}
-                         wxsqlite3 
+                         -lwxsqlite3-3.0 
                          ${LIBSSH_LIB} 
                          ${ADDITIONAL_LIBRARIES}
                          ${LIBUCHARDET_LIB})

--- a/wxcrafter/CMakeLists.txt
+++ b/wxcrafter/CMakeLists.txt
@@ -43,7 +43,6 @@
 
 # Include paths
 include_directories("${CL_SRC_ROOT}/Plugin" 
-                    "${CL_SRC_ROOT}/sdk/wxsqlite3/include" 
                     "${CL_SRC_ROOT}/CodeLite" 
                     "${CL_SRC_ROOT}/PCH" 
                     "${CL_SRC_ROOT}/Interfaces")
@@ -105,7 +104,7 @@
 
 if (NOT OFFICIAL_CL_BUILD)
     target_link_libraries(${PLUGIN_NAME} ${LINKER_OPTIONS} ${wxWidgets_LIBRARIES} 
-                          wxsqlite3 
+                          -lwxsqlite3-3.0 
                           ${SQLITE3_LIBRARY}
                           libcodelite 
                           plugin)
